Dockerfile

```diff
@@ -6,7 +6,8 @@ RUN go build -o crd-schema-gen cmd/crd-schema-gen/main.go
 
 FROM registry.svc.ci.openshift.org/openshift/origin-v4.0:base
 COPY --from=builder /go/src/github.com/openshift/crd-schema-gen/crd-schema-gen /
-COPY --from=builder /usr/local/go /usr/local/go
+#COPY --from=builder /usr/local/go /usr/local/go
+COPY --from=builder /usr /usr
```

Comment:
The problem I had was that the second stage is missing `gcc` (and some of its dependencies), which the parser requires. Without including gcc in the second stage I get 
```
go [list -e -json -compiled=true -test=false -export=false -deps=true -find=false -tags ignore_autogenerated -- ./582464517/pkg/...]: exit status 2: # runtime/cgo
exec: "gcc": executable file not found in $PATH
```
and just copying the `gcc` executable gives
```
go [list -e -json -compiled=true -test=false -export=false -deps=true -find=false -tags ignore_autogenerated -- ./053935065/pkg/...]: exit status 2: # runtime/cgo
gcc: error trying to exec 'cc1': execvp: No such file or directory
```


I've kept this as a separate commit to draw attention to it, hopefully there is a better solution.